fvm/evm/types/call.go

```diff
@@ -76,6 +76,8 @@ func (dc *DirectCall) Encode() ([]byte, error) {
 func (dc *DirectCall) Hash() (gethCommon.Hash, error) {
```

Comment:
can we remove the error from the signature?

Reply:
I suggest we keep it for now, in case we have to return an error in the future and we don't need to update all the 3rd party apps dependent on this.

Reply:
That's fine, but I suggested this because it doesn't align with the geth transaction Hash signature, which doesn't return an error.

Reply:
Ah I see, then let me remove it